src/relay/op/tensor/transform.cc

```diff
@@ -47,6 +47,66 @@ namespace tvm {
 namespace relay {
 using tir::IntImmNode;
 
+int64_t* ToVector(const runtime::NDArray& array) {
+  size_t len = array.Shape().front();
+  int64_t* rel_vec = new int64_t[len];
```

Comment:
Please don't use raw pointer. it will cause memory leak.

Reply:
Also consider move this function and [similar one](https://github.com/apache/incubator-tvm/blob/master/src/relay/op/tensor/transform.cc#L450) to some util.h so that others can reuse.

Reply:
Actually I think we don't need ```ToVector``` anymore since ```ToScalar``` can fully cover and have more supported types. I'll move ```ToScalar``` to ```pattern_util.h```.